SYCL/AOT/multiple-devices.cpp

```diff
@@ -10,6 +10,9 @@
 // UNSUPPORTED: cuda
 // CUDA is not compatible with SPIR.
 
+// The test is failing with GPU RT 30.0.100.9667
```

Comment:
The test also checks combination without GPU at all and it get's disabled because of unrelated GPU fail. I can't come up with combinations of options that allows to disable concrete RUN lines on concrete OS. We might consider re-arranging this test.

Reply:
The test will run only on a machine with CPU+GPU+Accelerator available,
As for my understanding, it is built for all target devices in the single fat binary. Then fat binary is run for every target device to check that every device image is functional.

The test is executed only if all devices are avaialble. No need to disable particualr device.

Reply:
> As for my understanding, it is built for all target devices in the single fat binary. Then fat binary is run for every target device to check that every device image is functional.

Yes, it is done on lines 18-22. On next lines device code is compiled manually by each of AOT compilers and different combinations of resulting binaries are supplied to a fat binary via -fsycl-add targets option. There is only CPU&FPGA run starting on line 51.

Reply:
To address that it we can split the test in separate PR.